spec/requests/invites_spec.rb

```diff
@@ -0,0 +1,52 @@
+require 'helpers/rails_helper'
+require 'helpers/api_helper'
+
+RSpec.describe "Invites", type: :request do
+  include_context "with authenticated requests"
+  include_context "with JSON responses"
+
+  let(:student) { create(:student) }
+  let(:teacher) { create(:teacher) }
```

Comment:
I would prefer that we create the teacher outside of the course, and set the teacher for the course when we create the course. More visibility outside of the factory.